hddm/utils.py

```diff
@@ -534,26 +578,27 @@ def qp_plot(x, groupby=None, quantiles=(0.1, 0.3, 0.5, 0.7, 0.9), ncols=None, dr
 
     return ax
 
+    # AF-Q: UNUSED?
     #create axes for subjects
-    n_subjs = len(data.subj_idx.unique())
```

Comment:
this happens after the `return`? Then yeah, remove.

Reply:
done